internal/pkg/config/v1/mock.go

```diff
@@ -220,3 +222,11 @@ func setUpConfig(conf *Config, ctx *Context, platform *Platform, credential *Cre
 		panic(err)
 	}
 }
+
+func SetTempHomeDir() {
```

Comment:
Are there any other areas of the codebase that could use this function? Run a global search for `os.Setenv("HOME"`?